src/triage/component/postmodeling/contrast/utils/aux_funcs.py

```diff
@@ -0,0 +1,58 @@
+'''
+Auxiliary functions and helpers:
+
+This set of functions are helper functions to format data 
+(i.e., prediction matrices, etc.) for plotting. This functions
+are called in both Model class and ModelGroup class in 
+evaluation.py.
+'''
+
+from sqlalchemy import create_engine
+from sqlalchemy.sql import text
+from collections import namedtuple
+import pandas as pd
+import yaml
+
+def create_pgconn(credentials_yaml):
+    '''
+    Create SQL connection object using a psycopg2 cursor and abiding to new
+    dssg/dsapp db user configuration. 
+
+    Arguments:
+        - credentials_yaml: .yaml file with db credentials
+    '''
+    with open(credentials_yaml) as f:
+        configs = yaml.load(f)
+    try:
+        conn = create_engine("postgresql://{user}:{password}@{host}/{dbname}".format(**configs))
+    except:
+        print("Error connecting to db.")
+
+    return conn 
+
+def get_models_ids(audited_model_group_ids, conn):
```

Comment:
I don't think this is used -- is it necessary at all?

Reply:
Also used in notebook